integrations/fullstory/lib/index.js

```diff
@@ -96,6 +99,34 @@ FullStory.prototype.identify = function(identify) {
   }
 };
 
+/**
+ * Page.
+ *
+ * @api public
+ * @param {Page} page
+ */
+
+FullStory.prototype.page = function(page) {
+  var category = page.category();
+  var name = page.fullName();
+  var opts = this.options;
+
+  // all pages
+  if (opts.trackAllPages) {
```

Comment:
Would be possible to reverse the order of this operation, as:
1- Check if it's a named page, then track it.
2- If not, check if it's a categorized page, then track it.
3- If not, check if all pages setting is enabled, then track it.
With this, we are sure we only send one page call. what do you think?

Reply:
Changed, this will eliminate duplicate page track calls.